classes/Infrastructure/Auth/SentinelAccountManagement.php

```diff
@@ -95,15 +95,15 @@ public function create($email, $password, array $data = []): UserInterface
      *
      * @return bool
      */
-    public function activate($email): bool
+    public function activate(string $email): bool
```

Comment:
Shall we stop returning something from this method and remove the return type declaration?

Alternatively, we could pull up the return type declaration into the interface. Personally, though, I would like to avoid returning a `bool` from a method to indicate it was invoked successfully.

Reply:
The reason I am asking that I think we should rely on the interface, not the implementation.